types/branca/branca-tests.ts

```diff
@@ -1,4 +1,4 @@
-const key = 'supersecretkeyyoushouldnotcommit';
+const key = '7ed049e344f73f399ba1f7868cf9494f4b13347ecce02a8e463feb32507b73a5';
```

Comment:
Why did you commit the key?

Reply:
This is not a *real* key, its a sample 32 byte hex string, the same sample string used in the upstream repository.

You cannot use just any string as the key (as was used previously). It *must* be a 32 byte (64 char) hex string, or a 32 byte `Buffer`. If it doesn't match this shape the init will fail.

Reply:
See the sample usage of the same key in the project README.md

https://github.com/tuupola/branca-js#secret-key

Reply:
Oops my fault, sorry!

Reply:
Thanks for merging. Cheers.